src/pulse3D/peak_detection.py

```diff
@@ -57,8 +57,8 @@ def peak_detector(
     max_time = time_signal[-1]
     start_time = np.max([0, start_time])
     end_time = np.min([end_time, max_time / MICRO_TO_BASE_CONVERSION])
-
     # how should we handle this?
+
```

Comment:
Not sure who left this comment but we might want to turn it into a TODO so it's more visible

Reply:
Kristian, added a TODO